src/index.tsx

```diff
@@ -157,9 +148,42 @@ var saveToWebmarkFolder = (url: string | undefined, title: string | undefined):
     );
 }
 
-var getRandomUrlFromFolder = (): string => {
-    //TODO: implement getRandomUrlFromFolder
-    return 'https://example.com/random';
+let getRandomUrlFromFolder = (): void => {
+    chrome.storage.sync.get(
+        ['webmarkFolderId'],
+        (result?) =>{
+            console.log('webmarkFolderId found.');
+            let webmarkFolderId: string = result!['webmarkFolderId'];
+            chrome.bookmarks.getSubTree(
+                webmarkFolderId,
+                (bookmarkTreeNodes) =>{
+                    let urlList : Array<string> = [];
```

Comment:
FYI: Just googled the difference between `Array<string>` and `string[]` out of curiosity.
https://stackoverflow.com/questions/36842158/arraytype-vs-type-in-typescript
Mark this as resolved once you read it.